.github/workflows/bump-helm-versions.yaml

```diff
@@ -0,0 +1,48 @@
+name: Bump Helm Chart Versions
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 1" # Run every Monday at 00:00 UTC
+
+env:
+  team_reviewers: 2i2c-org/tech-team
+
+jobs:
+  bump-helm-versions:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # For each new helm chart to bump the subchart versions of, add a new item in
+          # this matrix.
+          - name: "binderhub"
+            chart_path: "helm-charts/binderhub/Chart.yaml"
+            # To bump multiple subchart versions, provide the chart name and a URL to
+            # a published list of versions in this dictionary.
+            chart_urls: '{"binderhub": "https://raw.githubusercontent.com/jupyterhub/helm-chart/gh-pages/index.yaml"}'
```

Comment:
```suggestion
            chart_urls: '{"binderhub": "https://jupyterhub.github.io/helm-chart/index.yaml"}'
```

Reply:
The first URL is correct. It is a lot easier to parse [this file](https://github.com/jupyterhub/helm-chart/blob/gh-pages/index.yaml)  through ruamel than it is to scrape the URL you suggested with BeautifulSoup.